documentation/devsetup.md

```diff
@@ -11,20 +11,20 @@ These can be installed on your own compute but we suggest using one of the provi
 #### Prequisites
 
 **.NET Core**
-Nether is built on top of .NET Core 1.1. Install from [https://dot.net/core](https://dot.net/core). This is included in Visual Studio 2017. To install on Linux/OSX see the [RC4 release notes](https://github.com/dotnet/core/blob/master/release-notes/rc4-download.md)
+Nether is built on top of .NET Core 1.1. Make sure you install at least version 1.1 RC4 from [https://github.com/dotnet/core/blob/master/release-notes/download-archive.md](https://github.com/dotnet/core/blob/master/release-notes/download-archive.md). This is included in Visual Studio 2017. To install on Linux/OSX see the [RC4 release notes](https://github.com/dotnet/core/blob/master/release-notes/rc4-download.md)
 
 **SQL Server**
 The out-of-the-box configuration for nether uses in-memory data stores. You can configure it to run against SQL Server as per [the configuration docs](./configuration.md), and there is a partial implementation of MongoDB providers
 
 **Azure Powershell**
-Nether has been designed to take advantage of various services that are part of the WIndows Azure platform. This includes but is not limited to hosting the REST APIs that are Nether. As such, our default "production" deployment scripts are based on Azure and currently require use of the PowerShell cmdlets for Azure.
+Nether has been designed to take advantage of various services that are part of the Microsoft Azure platform. This includes but is not limited to hosting the REST APIs that are Nether. As such, our default "production" deployment scripts are based on Azure and currently require use of the PowerShell cmdlets for Azure.
 
 Azure Powershell is available via the [Web Platform Installer](https://www.microsoft.com/web/downloads/platform.aspx) or by issuing the command "install-module AzureRM" from the PowerShell command line. For more about the Azure Powershell Cmdlets, please see the [official documentation](https://docs.microsoft.com/en-us/powershell/).
 
 **Unity SDK (optional)**
 We support and plan to build SDKs for [Unity](http://unity3d.com). However, since Nether is exposed as a series of easily consumable REST API's, it is accesible by almost any platform/language.
 
-## User Authentication
+## Gamer Authentication
```

Comment:
But the admin won't need to setup the FB authentication, correct?
Should there be a paragraph just for admin?